src/locales/en.json

```diff
@@ -795,7 +795,7 @@
             "ButtonUnlock": "Unlock",
             "Bypass": "Bypass",
             "Cancel": "Cancel",
-            "ClogDetection": "Clog Detection",
+            "ClogDetection": "Clog/Tangle Detection",
```

Comment:
pls do not update old translations. use a new key for a different translation. i would also not update this line in this PR, because this is important for the flowmeter PR.